duckduckgo/BrowserServicesKit
# Update TrackerRadarKit

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/0/1201752704214881/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1161
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/592
What kind of version bump will this require?: Patch

**Optional**:

Tech Design URL:
CC:

**Description**:
This PR updates TrackerRadarKit which now includes support for load types in CBR creation. In other terms this will provide better blocking for CNAMEd trackers which appear in a 1st party context.

**Steps to test this PR**:
1. Update TDS url to use non-apple list (this will include CNAMEs)
2. Visit a site with CNAMEd trackers. Ensure the trackers are blocked



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
